tests/installation/partitioning/modify_existing_partition.pm

```diff
@@ -21,9 +21,10 @@ sub run {
     my $test_data   = get_test_suite_data();
     my $partitioner = $testapi::distri->get_expert_partitioner();
     $partitioner->run_expert_partitioner();
+    record_info("Resize root");
     $partitioner->resize_partition_on_gpt_disk($test_data->{root});
     for my $part (keys %$test_data) {
-        record_info("modify $part", "$$test_data{$part}->{existing_partition}");
+        record_info("Handle $part", "$$test_data{$part}->{existing_partition}");
```

Comment:
For the name of the screen, better "Edit partition" instead of "Handle", it is more clear, wdyt?

Reply:
Well, in the case of swap, we only enable it. I ould not find the perfect verb to describe "modify or enable $part" with less than two words...  But yeah, maybe "Edit" is better

Reply:
Also double `$$` is not needed, just one (it was existing code)

Reply:
> Also double `$$` is not needed, just one (it was existing code)

Without it, it fails: http://waaa-amazing.suse.cz/tests/10793

Reply:
I see, cannot interpolate `$part` probably in that case, perhaps:
"$test_data{\$part}->{existing_partition}" (I'm not sure) but I was referring without double quotes surrounding in case you didn't try that:
`record_info("Handle $part", "$$test_data{$part}->{existing_partition}");`